draft-secure-credential-transfer.md

```diff
@@ -211,6 +213,30 @@ Sender device may terminate the secure credential transfer by deleting the mailb
 ~~~
 {: #stateful-flow-image title="Sample stateful workflow"}
 
+## Credential Vertical in share url
+
+When a user interacts with a share url on a Receiver device it can be helpful to know what Credential Vertical this share is for. This is particularly important if the Receiver device has multiple apps that can handle a share url. For example, a Receiver device might want to handle a general access share in their wallet app, but handle car key shares in a special car app.
```

Comment:
```suggestion
When a user interacts with a share URL on a Receiver device it can be helpful to know what Credential Vertical this share is for. This is particularly important if the Receiver device has multiple applications that can handle a share URL. For example, a Receiver device might want to handle a general access share in their wallet app, but handle car key shares in a specific car application.
```